tests/repositories/test_experiments.py

```diff
@@ -91,3 +91,86 @@ def test_store_and_load_experiment():
 
         # with those adjustments -- assert the stored data matches the loaded
         assert loaded_experiment == experiment
+
+
+def _store_experiment(conn, path):
```

Comment:
This cannot be directly used in the existing `test_store_and_load` function -- as it doesn't track some variables and do some assertions. If we _really_ wanted to it could be made to work but I think it's better to keep the two processes separate.

I'm also open to the comment "this is too lazy of a way to prep data for testing" but it was easy and it seems to work OK.